style.css

```diff
@@ -25,70 +41,77 @@ section {
 }
 
 h2 {
+  font-size: 2rem;
   margin-bottom: 20px;
 }
 
-/* ----------App.js---------- */
+/* ----------cyf---------- */
 
-.App {
-  background-color: #f0f0f0;
+body {
+  background-color: var(--gray);
   min-height: 100vh;
+  min-width: 300px;
 }
 
-.App .cyf {
-  background-color: #fff;
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+.cyf {
+  background-color: var(--white);
+  box-shadow: 0 2px 4px var(--shadow);
   border-radius: 5px;
-  margin-top: 20px;
-  padding: 20px;
+  margin-top: 1rem;
+  padding: 1rem;
 }
 
-.App .cyf .about-trainees {
-  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
-  background-color: #2f363e;
-  color: #fff;
-  border-radius: 8px;
-  padding-bottom: 50px;
-}
-
-.App .cyf .about-trainees figure {
-  display: grid;
-  grid-template-columns: 1fr;
-  gap: 20px;
-}
-
-/* ----------Header.jsx---------- */
+/* ----------Header---------- */
 
 .header {
   display: flex;
   flex-direction: column;
   align-items: center;
+  padding: 0 1rem;
 }
 
 .header h1 {
   flex-grow: 0.75;
+  font-size: 2rem;
```

Comment:
The font scale seems a bit random.

Typically in a design we define a font scale  - perhaps put this at the top of the page with your colour palette and your spacing - and then pick sizes from that scale. 

https://fonts.google.com/knowledge/glossary/scale